next/content/docs/guides/vercel.md

```diff
@@ -1,73 +1,47 @@
+This guide describes how to integrate Neon with Vercel using the Neon integration from the Vercel marketplace. The integration automatically configures the required environment variables in your Vercel project.
 
-To create a Neon project and access it from a Next.js application:
+## What the Neon integration does
 
-1. [Create a Neon project](#create-a-neon-project)
-2. [Create a Next.js project](#create-a-nextjs-project)
-3. [Add a PostgreSQL client to your app](#add-a-postgresql-client-to-your-app)
-4. [Add your Neon connection details](#add-your-neon-connection-details)
-5. [Connect to the Neon database](#connect-to-the-neon-database)
+TBD
 
-## Create a Neon project
+## Prerequisites
 
-If you do not have one already, create a Neon project. Save your connection details including your password. They are required when defining connection settings.
+- An existing Neon project. If you do not have a Neon project, refer to the [Setting up a project](../../get-started-with-neon/setting-up-a-project/) to get started.
+- A [Vercel account](https://vercel.com).
+- A Vercel project. For Vercel project creation instructions, see [Projects](https://vercel.com/docs/concepts/projects/overview), in the Vercel documentation.
 
-1. Navigate to the [Projects](https://console.neon.tech/app/projects) page in the Neon Console.
-2. Click **New Project**.
-3. Specify a name, a PostgreSQL version, a region, and click **Create Project**.
+## Deploy the integration
 
-For additional information about creating projects, see [Set up a project](/docs/get-started-with-neon/setting-up-a-project).
+1. Navigate to the [Neon Vercel integrations page](https://vercel.com/integrations/neon) and click **Add integration**.
+1. Select a Vercel account to add the integration to and click **Continue**.
+1. Select the projects to which the integration will be added. You can select **All Projects** or a **Specific Project**. After you have made your selection, click **Continue**.
+1. Review the required permissions, and click **Add Integration** to continue.
+1. In the **Integrate Neon** dialog:
+    1. Select the Vercel project that you want to add the integration to, and click **Continue**. If the Vercel project is already connected to a Neon project, continuing the integration replaces the existing configuration.
+    1. Select a Neon project, a database, and role that Vercel will use to connect. The Neon Free Tier supports a single project per user. If you do not have a Neon project, you can create one. Click **Connect**.
+    1. The integration sets two `DATABASE_URL` variables in Vercel, one for your production database, and one for database branches that are created for preview deployments. The production database resides on the `main` branch of your Neon project. The database branches created for preview deployments are branched from `main`. Click **Confirm** to accept the environment variable settings.
```

Comment:
in the near future the integration will also be setting environment variables for the development environment. we can mention this as coming soon.

Reply:
This is now mentioned